.github/workflows/deploy-kotlin.yml

```diff
@@ -130,7 +141,7 @@ jobs:
   test:
     name: Test
     needs: initialize
-    if: always() && needs.initialize.result == 'success'
```

Comment:
The always() here was a bit redundant, as the clause also contains a check for the initialize jobs result status